libdnf/rpm/sack_impl.hpp

```diff
@@ -20,13 +20,39 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #ifndef LIBDNF_RPM_SACK_IMPL_HPP
 #define LIBDNF_RPM_SACK_IMPL_HPP
 
+#include "solv/map.hpp"
+
 #include "libdnf/rpm/package.hpp"
 #include "libdnf/rpm/sack.hpp"
 
 extern "C" {
 #include <solv/pool.h>
 }
 
+#include <vector>
+
+constexpr const char * SOLVABLE_NAME_ADVISORY_PREFIX = "patch:";
+constexpr size_t SOLVABLE_NAME_ADVISORY_PREFIX_LENGTH = 6;
```

Comment:
We need `SOLVABLE_NAME_ADVISORY_PREFIX_LENGTH` constant?
Anyway compiler (during compile time, no cost during runtime) can compute the length. There is no need to hardcode the value by hand:
`constexpr size_t SOLVABLE_NAME_ADVISORY_PREFIX_LENGTH = std::char_traits<char>::length(SOLVABLE_NAME_ADVISORY_PREFIX);`